src/community/geopkg/src/main/java/applicationContext.xml

```diff
@@ -7,19 +7,15 @@
 <!DOCTYPE beans PUBLIC "-//SPRING//DTD BEAN//EN" "http://www.springframework.org/dtd/spring-beans.dtd">
 <beans>
 
-    <bean id="geopkgGetMapOutputFormat" class="org.geoserver.geopkg.GeoPackageGetMapOutputFormat">
+    <bean id="geopkgGetMapOutputFormatWPS" class="org.geoserver.geopkg.wps.GeoPackageGetMapOutputFormatWPS">
```

Comment:
So by pluggin this one in, GeoServer will find two of them... does that cause any misbehavior?
Why not just depend on the geopkg-output module?

Reply:
Different bean, this one makes the WPS output format.

Reply:
Perhaps I did not understand, is WPS output format and WFS Output format implementing the same interface?

Reply:
This bean that you are re-declaring in the community module is a WMS output format. The GeoPackage process generates tiles entries calling onto the WMS output format. The way it's set up, you have two classes for the WMS output format, and two beans that declare them. They are related by inheritance, the WPS one extends the WMS one.